definitions/divorce/json/CaseField/CaseField-alt-service-process-server-nonprod.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "LiveFrom": "05/11/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ProcessServerServiceCaseField",
```

Comment:
Could you please name this field more descriptively?

Reply:
How does `ProcessServerServiceDecision` sound?

Reply:
Sounds good.